spec/Section 3 -- Type System.md

```diff
@@ -1923,3 +1929,62 @@ type ExampleType {
   oldField: String @deprecated(reason: "Use `newField`.")
 }
 ```
+
+### @defer
+
+```graphql
+directive @defer(label: String, if: Boolean) on FRAGMENT_SPREAD | INLINE_FRAGMENT
+```
+
+The `@defer` directive may be provided for fragment spreads and inline fragments to 
+inform the executor to delay the execution of the current fragment to indicate 
+deprioritization of the current fragment. A query with `@defer` directive will cause
+the request to potentially return multiple responses, where non-deferred data is 
+delivered in the initial response and data deferred is delivered in a subsequent
+response. `@include` and `@skip` take precedence over `@defer`. 
+
+```graphql example
+query myQuery($shouldDefer: Boolean) {
+   user {
+     name
+     ...someFragment @defer(label: 'someLabel', if: $shouldDefer) 
+   }
+}
+fragment someFragment on User {
+  id
+  profile_picture {
+    uri
+  }
+}
+```
+
+#### @defer Arguments
+
+* `if: Boolean` - When true, fragment may be deferred. If omitted, defaults to `true`.
```

Comment:
Does this argument determine if the fragment/field is deferred or whether it is included in the actual result? In other words, is this just shorthand for not having to add `@skip`/`@include` directives to the fragment/field? After reading through the graphql-js PR I’m pretty sure this argument just makes the actual deferring/streaming optional while the fragment continues to appear in the result, but it would be nice to have even greater clarity on this.

Reply:
Just whether deferred

https://github.com/graphql/graphql-js/blob/1012e3e481d3b03ab92fb59a05b73a8b867f13d5/src/execution/__tests__/defer-test.js#L111

Reply:
Yep just wished it was a little clearer in the spec as well, because it’s likely to be a source of confusion.

Reply:
@brainkim I added some more details to hopefully clear this up.